ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/Job_management.java

```diff
@@ -0,0 +1,342 @@
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ****************************************************************************
+ */
+
+package com.spectralogic.ds3client.integration;
+
+import com.google.common.collect.Lists;
+import com.spectralogic.ds3client.Ds3Client;
+import com.spectralogic.ds3client.commands.*;
+import com.spectralogic.ds3client.commands.spectrads3.*;
+import com.spectralogic.ds3client.helpers.Ds3ClientHelpers;
+import com.spectralogic.ds3client.helpers.FileObjectPutter;
+import com.spectralogic.ds3client.helpers.JobRecoveryException;
+import com.spectralogic.ds3client.helpers.options.WriteJobOptions;
+import com.spectralogic.ds3client.models.*;
+import com.spectralogic.ds3client.models.bulk.Ds3Object;
+import com.spectralogic.ds3client.networking.FailedRequestException;
+import com.spectralogic.ds3client.serializer.XmlProcessingException;
+import com.spectralogic.ds3client.utils.ResourceUtils;
+import javafx.util.converter.DateTimeStringConverter;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.net.URISyntaxException;
+import java.nio.channels.FileChannel;
+import java.nio.channels.SeekableByteChannel;
+import java.nio.file.*;
+import java.security.SignatureException;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+import java.util.UUID;
+
+import static com.spectralogic.ds3client.helpers.Ds3ClientHelpers.*;
+import static com.spectralogic.ds3client.integration.Util.*;
+import static org.hamcrest.CoreMatchers.*;
+import static org.junit.Assert.*;
+import static org.junit.Assume.assumeThat;
+
+public class Job_management {
+
+    private static final Logger LOG = LoggerFactory.getLogger(Job_management.class);
+
+    private static Ds3Client client;
+
+    @BeforeClass
+    public static void startup() {
+        client = fromEnv();
+    }
+
+    @AfterClass
+    public static void teardown() throws IOException {
+        client.close();
+    }
+
+    @Test //also in smoke test--
+    public void modifyJobPriority() throws IOException, SignatureException, XmlProcessingException {
+        final String bucketName = "test_modify_job_priority";
+        try {
+            client.putBucket(new PutBucketRequest(bucketName));
+
+            final List<Ds3Object> objects = new ArrayList<>();
+            final Ds3Object obj = new Ds3Object("test", 2);
+            objects.add(obj);
+
+            final WriteJobOptions jobOptions = WriteJobOptions.create().withPriority(Priority.LOW);
+
+            final Ds3ClientHelpers.Job job =
+                    wrap(client).startWriteJob(bucketName, objects, jobOptions);
+
```

Comment:
I'm not sure I'm clear on that one--because sometimes there are options, sometimes not, and the objects vary.